dist/script.js

```diff
@@ -1,15 +1,24 @@
 (function(){'use strict';
 
-/** *****************************************************************************
+/** ************************************************************************
  * FORGEROCK | AM DEVICE MATCH SCRIPT
- * This is a generated file. Do not directly modify.
+ * 
+ * script.js
+ * 
+ *  Copyright (c) 2020 ForgeRock. All rights reserved.
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ * 
+ * *************************************************************************
+ * 
+ * THIS IS A GENERATED FILE. Do not directly modify.
```

Comment:
You can ignore changes below this line. It is a built file and upon updating Babel and Rollup, there were some internal changes to how it writes the wrapper functions into this file. All tests pass, so there's no change in external functionality.